setup.py

```diff
@@ -60,6 +60,9 @@
     "python-dateutil",
     "certifi>=2024.07.04",
     "Events",
+    # License: Apache 2.0
+    # gRPC transport client libraries
+    "opensearch-protobufs==0.19.0",
```

Comment:
Do we need to lock this in at this version? Or can we simply point to >=?. Im wondering what happens when we bump protobufs, do we need to manually change versions here?